index.js

```diff
@@ -94,7 +94,9 @@ function is(x) {
         "דרייַצן", // Yiddish,
         "דרייצן", // Yiddish (without diacritics),
         "kumi na tatu", // Swahili
-
+        "mười ba", // full Vietnamese
+        "muoi ba", // simplied Vietnamse
+        
```

Comment:
haha I'm gonna do the same thing :) 

I suggest we can make the comments following people: 

```js
// Vietnamese 
// Vietnamese (without tones)
```

I intended to add something like `mười pa` lol